package.json

```diff
@@ -5,52 +5,76 @@
   "main": "src/app.ts",
   "scripts": {
     "start": "ts-node -r tsconfig-paths/register src/app.ts --type-check",
+    "start:debug": "yarn start --debug",
+    "start:runtime": "TS_NODE_PROJECT=tsconfig.runtime.json node -r tsconfig-paths/register -r source-map-support/register build/app",
```

Comment:
Is this for production? The production should build the source into Javascript code and use pm2.

Reply:
sure, just add [README.md](https://github.com/visuallylab/backend-boilerplate/pull/6/files#diff-1).
Please review and give me some feedback~~if any not understand.